corehq/apps/case_search/filter_dsl.py

```diff
@@ -151,6 +152,8 @@ def _raise_step_RHS(node):
     def _unwrap_function(node):
         """Returns the value of the node if it is wrapped in a function, otherwise just returns the node
         """
+        if isinstance(node, UnaryExpression) and node.op == '-':
```

Comment:
A `UnaryExpression` is only ever a `-`:

https://github.com/emory-libraries/eulxml/blob/17d71c7d98c0cebda9932b7f13e72093805e1fe2/eulxml/xpath/ast.py#L77-L79